### ISSUE TITLE
Review site for accessilibility issues.

### ISSUE THREAD

**Issue by rz1005** [exewebdev/exe#35]
We need to look at all files for any accessibility issues.

**Comment by misspokeycat** [comment]
Autonomous checking for this is possible via http://tenon.io/ - seems like a good place to start.

**Comment by misspokeycat** [comment]
This is not a very good bug as it is not specific - what is the exit criterion on this?

**Comment by rz1005** [comment]
It's not a bug it's an issue (task). Something that needs to be looked at.  We either need to find a library that we do this for us automatically (preferable) or we make a llst and do it manually.

**Comment by misspokeycat** [comment]
Our task runner is Grunt, so this https://www.npmjs.com/package/grunt-accessibility might work?

**Comment by rz1005** [comment]
I will take a look at it.

**Comment by misspokeycat** [comment]
In commit 422a5a7 I added the accessibility checker to the Grunt tests, configured to fail the build if there are any accessibility errors.

(I also explained how to run the tests locally, in case you ever failed the build and wondered why your changes were not reflected on Heroku :smiley:)

Because it is now a requirement that all code must pass accessibility checks before reaching production, I will close this issue, as there was no real exit criterion specified, and the site validates according to `grunt-accessibility` to WCAG2A accessibility standards without errors.
